repos/system_upgrade/common/libraries/module.py

```diff
@@ -16,7 +16,9 @@
 def _create_or_get_dnf_base(base=None):
     if not base:
         base = dnf.Base()
+        base.init_plugins()
         base.read_all_repos()
+        base.configure_plugins()
```

Comment:
@Rezney  To make it more clear for people, why the plugins are not configured immadiately after the initialisation.
```suggestion
        # configure plugins after the repositories are loaded
        # e.g. the amazon-id plugin requires loaded repositories
        # for the proper configuration.
        base.configure_plugins()
```